docs/backups-ondemand.md

```diff
@@ -0,0 +1,50 @@
+# Making on-demand backup
+
+1. To make an on-demand backup, you should first check your Custom Resource for
+    the necessary options and make changes, if needed, using the
+    `deploy/cr.yaml` configuration file:
+
+    * the `backup.enabled` key should be set to `true`,
+
+    * `backup.storages` subsection should contain at least one [configured storage](backups-storage.md).
+
+    You can apply changes in the `deploy/cr.yaml` file with the usual
+    `kubectl apply -f deploy/cr.yaml` command.
+
+2. Now use *a special backup configuration YAML file* with the following
+    keys:
+
+    * `metadata.name` key should be set to the **backup name**
+        (this name will be needed later to [restore the bakup](backups-restore.md)),
+
+    * `spec.clusterName` key should be set to the name of your cluster (prior to
+        the Operator version 1.12.0 this key was named `spec.psmdbCluster`),
+
+    * `spec.storageName` key should be set to the name of your [already configured storage](backups-storage.md).
+
+    * optionally you can set the `spec.type` key to `physical` if you would like
+       to make physical backups instead of logical ones (please see the
+       [physical backups limitations](backups.md#physical)). Otherwise set
```

Comment:
We seem to link to a file which is being deleted?

Reply:
We restored this file several commits ago :)